lib/pbench/server/api/__init__.py

```diff
@@ -26,6 +26,7 @@
 from pbench.server.api.resources.query_apis.index_mappings import IndexMappings
 from pbench.server.api.resources.query_apis.datasets_publish import DatasetsPublish
 from pbench.server.api.resources.query_apis.month_indices import MonthIndices
+from pbench.server.api.resources.query_apis.search_api import IndexSearchAPI
```

Comment:
None of the other classes use this `API` suffix; I think it's unnecessary (e.g., the path is `api.resources.query_apis` so we already have "API" twice in the string; we don't really need it three, or four. The file name also breaks the pattern established in the others, and the inconsistency bugs me (and will probably bug me more over time as I guess wrong). That's why I changed `tracker.py` to `dataset.py` ... let's not make the same mistake again.

(Arguably, `query_apis` was already unnecessarily redundant and I should have just gone with `pbench.server.api.resources.queries` ... however, the directory path is sort of a singleton pattern, while the class modules within the directory constitute a matched set which ought to be predictable.)

```
from pbench.server.api.resources.query_apis.index_search import IndexSearch
```